workloads/upgrade-perf/README.md

```diff
@@ -0,0 +1,94 @@
+# Upgrade Scripts
+
+The purpose of the upgrade scripts is to upgrade an Openshift Cluster
+to a given version. It will report the timings to Elasticsearch.
+
+Running from CLI:
+
+```sh
+$ ./run_<test-name>_fromgit.sh 
+```
+
+## Environment variables
+
+### VERSION
+Default: ''
+The target version of the cluster. THIS IS A REQUIRED VARIABLE
+
+## TOIMAGE
+Default: ''
+This is an optional location of an image to upgrade to. If set you will STILL need to assign
+the VERSION variable as well to match what the end result will be.
+
+### POLL_INTERVAL
+Default: 5
+How long (in seconds) to have the test wait inbetween ready checks
+
+### TIMEOUT
+Default: 240
+Timeout value (in minutes) for the upgrade. NOTE: there is no rollback on failure
+
+### ES_USER
+Default: `` 
+Username for elasticsearch instance
+
+### ES_PASSWORD
+Default: `` 
+Password for elasticsearch instance
+
+### ES_SERVER
+Default: `milton.aws.com`  
+Elasticsearch server to index the results of the current run
+
+### ES_PORT
+Default: ``  
+Port number for elasticsearch server
+
+### COMPARE (not implemented yet)
```

Comment:
We might want to remove the uperf reference.

Reply:
HA! Good catch. Also fixed in the in scale doc since its a copy of a copy.